backend/README.md

```diff
@@ -1,4 +1,20 @@
 ## LexBox backend
 
-This section is for the backend of the LexBox project. 
-The backend is a dotnet core api. With a proxy to route requests for hg and resumable.
\ No newline at end of file
+The LexBox backend is a dotnet core api. With a proxy to route requests for hg and resumable.
+
+### ASP.NET Routes
+
+| Route | Description | Notes |
+|---|---|---|
+| **Send/Receive** | |
+| `/{project-code}`<sup>1</sup> OR `/hg/{project-code}` | hg S/R  | <sup>1</sup>Obviously it would be a bit problematic if a `project-code` were `api`, but that's the URL baked into Chorus clients around the world 🤷. (Do we still need this or could our proxies send our hg domains to `/hg`?) |
```

Comment:
sadly no, ingress has very limited support for rewriting requests which is what that would require.